consensus/polybft/consensus_runtime.go

```diff
@@ -134,6 +134,7 @@ func newConsensusRuntime(log hcf.Logger, config *runtimeConfig) (*consensusRunti
 		lastBuiltBlock:     config.blockchain.CurrentHeader(),
 		proposerCalculator: proposerCalculator,
 		logger:             log.Named("consensus_runtime"),
+		stateSyncManager:   &dummyStateSyncManager{},
```

Comment:
Let's instantiate stateSyncManager only once. Create some function which would check if bridge is enabled and based on it would return either `nilStateSyncManager` or (proper) `StateSyncManager` instance.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1067/commits/562b685df4cc5a3b75eb492d9ee85d1e524c1f8b